Source/WebKit/UIProcess/Network/NetworkProcessProxy.cpp

```diff
@@ -1764,13 +1764,41 @@ void NetworkProcessProxy::processPushMessage(PAL::SessionID sessionID, const Web
     if (auto it = permissions.find(origin); it != permissions.end())
         permission = it->value ? PushPermissionState::Granted : PushPermissionState::Denied;
 
-    sendWithAsyncReply(Messages::NetworkProcess::ProcessPushMessage { sessionID, pushMessage, permission }, WTFMove(callback));
+    RefPtr<ProcessAssertion> assertion = ProcessAssertion::create(getCurrentProcessID(), "WebKit Process Push Event"_s, WebKit::ProcessAssertionType::UnboundedNetworking);
```

Comment:
Could use `auto`

Reply:
ProcessAssertion::create makes a Ref.
The idea here is move that to a RefPtr so it's explicitly nullable.

Reply:
Oh, I see 👍🏻